src/components/pages/team/team-details/team-details.jsx

```diff
@@ -6,8 +6,13 @@ import { useArrayHandler } from 'hooks/use-array-handler'
 import PersonAddIcon from '@material-ui/icons/PersonAdd'
 import { Button, Dialog, Avatar, List, ListItem, ListItemAvatar, ListItemText } from '@material-ui/core'
 import styles from './team-details.module.scss'
+import { joinTeam } from 'services/axios/auth-api-calls/teams'
 
 const TeamDetails = ({ data, open, onClose }) => {
+	async function onJoinTeamClick() {
```

Comment:
Since this function hasn't an await nor a try/catch it's not needed to be declared as async.

Reply:
Done in https://github.com/jamtasticgd/jamtastic/pull/37/commits/d78f63ba10355c7b4b1bce87fa32a3c205afe6f5